### DISCUSSION TITLE
Changing the login node name

### DISCUSSION THREAD

**Discussion by tdido** [GoogleCloudPlatform/cluster-toolkit#1792] [Q&A] [answered]
Hello.

Is it possible to define a custom name for a login node in a cluster created with e.g. the hpc-slurm example? Or at least remove the random string that gets inserted (e.g. `mycluster-login-glbm2wun-001`)?

Thanks!

**Comment by cdunbar13** [comment, marked as answer]
I don't believe this is possible with the v5 version of the Slurm-GCP modules, which is the latest we currently support.  Looking at the [slurm_login_instance.tf](https://github.com/SchedMD/slurm-gcp/blob/v5/terraform/slurm_cluster/modules/slurm_login_instance/main.tf) from SchedMD, it appears that the random suffix and "login" portion of the name is defined in their module, not the HPC-Toolkit.  The only customizable portion is the cluster name.

I will confirm with my team.

**Reply by nick-stroud** [reply]
Note that the slurm_cluster_name must be the same between the controller and login nodes.

**Reply by tdido** [reply]
Thank you for the comment. Please note that I wasn't really referring to changing the cluster name but just the login node name. It's the only one getting a random string inserted into its name, and it's a bit inconvenient since it's the one that the users use to access the cluster.

**Reply by tdido** [reply]
Just wanted to mention that the v6 login module allows setting a custom string using `name_prefix`, which will then be used instead of the random string.
